platforms/core-configuration/model-core/src/integTest/groovy/org/gradle/api/provider/MapPropertyIntegrationTest.groovy

```diff
@@ -383,6 +380,10 @@ task thing {
         '"${str.substring(0, 1)}"' | 'project.provider { "${str.toLowerCase().substring(1, 2)}" }'
     }
 
+    @Requires(
+        value = IntegTestPreconditions.NotConfigCached,
+        reason = "Test relies on modifying properties at execution time, but CC finalizes them before execution"
```

Comment:
❌ This is wrong. It's not that CC finalizes them before execution, the task scheduler itself finalizes the properties before letting the task execute. This test is relying on the fact that, without CC, the properties of the **unscheduled** task `verify` are not finalized.

Reply:
Thanks. Changed it to use an extension (as I think you suggested) instead. Please see: https://github.com/gradle/gradle/pull/27587